ffi/capi/src/bidi.rs

```diff
@@ -49,7 +49,7 @@ pub mod ffi {
             hidden
         )]
         #[diplomat::attr(not(supports = utf8_strings), disable)]
-        #[diplomat::attr(*, rename = "for_text")]
+        #[diplomat::attr(auto, rename = "for_text")]
```

Comment:
`rename` is not gated, this should either be `*` or `supports = utf8_strings`

Reply:
oops

Reply:
Might be worth tweaking diplomat to yell at you if you use `auto` uselessly